.github/workflows/tag_image_push_rhel.yml

```diff
@@ -182,6 +162,10 @@ jobs:
 
           wait_for_container_scan "$RHEL_PROJECT_ID" "$VERSION" "$RHEL_API_KEY" "$TIMEOUT_IN_MINS"
 
+      - name: Set MC version
+        run: |
+          echo "HZ_MC_VERSION=$(echo "${{ inputs.HZ_VERSION }}" | cut -d '.' -f 1,2)" >> $GITHUB_ENV
```

Comment:
Why `cut` over `semver-utils` as before?

Reply:
IDK :D
Less downloads for runner, still readable.
I don't have strong opinion on this, I can change back to `semver-utils` and concat 
`${{ steps.version.outputs.major }}.${{ steps.version.outputs.minor }}`
if you like that one more.

Reply:
> I don't have strong opinion on this

Me neither, but I thought I'd ask